docs/DeveloperGuide.adoc

```diff
@@ -366,6 +370,73 @@ _{More to be added}_
 == Use Cases
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+[discrete]
+=== Use case: Add person
```

Comment:
Maybe change "Add person" to "Add contact" instead? Just for some semantics and consistency, because for my use case "Restoring recently deleted person" sounds weird so I put "Restoring recently deleted contact". This applies to all other instances of "person".